app/lib/redis_configuration.rb

```diff
@@ -2,12 +2,17 @@
 
 class RedisConfiguration
   class << self
+    def establish_pool(new_pool_size)
+      @pool.shutdown { |redis| redis.close } if @pool
```

Comment:
As per CodeClimate's suggestions:

```suggestion
      @pool&.shutdown(&:close)
```